packages/gatsby-theme-docs/src/components/child-sections-nav.js

```diff
@@ -107,7 +107,7 @@ const Link = styled.a`
   font-size: ${designSystem.typography.fontSizes.small};
   color: ${designSystem.colors.light.textSecondary};
   text-decoration: none;
-  overflow-wrap: break-word;
+  overflow-wrap: anywhere;
```

Comment:
As an alternative, I've used `anywhere` here to break words if there are too long. Maybe you can give me some feedback here which solution you like more

Reply:
Good point, I think `anywhere` is the only value that works.